dali/test/python/test_pipeline_debug.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
At the first glance, what lacks from my perspective is a test, where the pipeline is configured on the instantiation level, e.g.:
```python
@pipeline_def
def simple_pipe():
  data = jpegs, labels = fn.readers.file(file_root=file_root, shard_id=0, num_shards=2)
  return data

pipe = simple_pipe(batch_size=8, num_threads=3, device_id=0, debug=True)
```

Reply:
done